### ISSUE TITLE
Color contrast issue with GitHub markdown styles used in terra-markdown for code elements inside of block-quotes

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#2188]
# Bug Report

## Description
When rendering `<code />` element inside of blockquotes with the GitHub markdown styles used in terra-markdown, the contrast between the code text and the background are insufficient in regards to accessible contrast ratios. 

## Steps to Reproduce

1. Navigate to https://engineering.cerner.com/terra-ui/#/components/terra-application-layout/application-layout/about
2. Run tooling to check contrast ratio on page

## Expected Behavior
No contrast issues

## Possible Solution
Add styles to set font color on code blocks in terra-markdown

## Environment

* Component Name and Version: terra-markdown

**Comment by bjankord** [comment]
Need to add the following CSS to this file: https://github.com/cerner/terra-core/blob/master/packages/terra-markdown/src/Markdown.scss
```css
.markdown-body blockquote {
  color: #24292e;
}
```
